packages/forms-web-app/src/dynamic-forms/validator/date-validator.js

```diff
@@ -107,11 +107,13 @@ class DateValidator extends BaseValidator {
 					if (req.body[dayInput]) {
 						return this.noMonthErrorMessage;
 					}
+					// todo: empty error message bug
 				}),
 			body(yearInput)
 				.notEmpty()
 				.withMessage((_, { req }) => {
 					if (req.body[dayInput] && req.body[monthInput]) return this.noYearErrorMessage;
+					// todo: empty error message bug
```

Comment:
what's the bug?

Reply:
https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3802

Reply:
probably a better way to resolve this, but it validates each field separately, 
But we only want 1 error if each is left empty, not 3
It adds all but just returns undefined for the message for month/year, leaving an empty link in the page that can be accessed with keyboard